.vscode/cscope_conf.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This file may not be needed, if so should be removed from the pull request.

Reply:
You're adding it here but then also adding it to .gitignore? One of these shouldn't be necessary